packages/flutter_tools/templates/plugin/windows.tmpl/pluginClassSnakeCase.cpp.tmpl

```diff
@@ -1,7 +1,7 @@
 #include "{{pluginClassSnakeCase}}.h"
 
 // This must be included before many other Windows headers.
-#include <windows.h>
+#include <Windows.h>
```

Comment:
```
Non-portable path to file '<Windows.h>'; specified path differs in case from file on disk [clang-diagnostic-nonportable-path-system-include-path]
```

It is from `ReSharper C++` plugin of Visual Studio. I guess lint tools would warn as well if `clang-diagnostic-nonportable-path-system-include-path` is flaged

Reply:
My understanding is that the capitalization of the header varies depending on the installation, in which case the warning is pretty much meaningless in this situation.